docs/scenarios/index.md

```diff
@@ -29,5 +29,5 @@ ms.assetid: 42c7ea39-8da5-499c-9a02-c1d27b37fc40
     * [Deploying ASP.NET web applications](deployment/aspnet-applications.md)
 
 * [Authoring Libraries for .NET Core RC2](library-authoring/index.md)
```

Comment:
You changed this title to use RTM instead of RC2 in toc.md, but not here or in the article itself.